Packs/CommonScripts/Scripts/FormattedDateToEpoch/FormattedDateToEpoch.py

```diff
@@ -1,17 +1,19 @@
 import demistomock as demisto
+import dateparser
 from datetime import datetime, timezone
+from typing import Optional
 
 
-def date_to_epoch(date, formatter):
+def date_to_epoch(date: str, formatter: Optional[str] = None):
     epoch = datetime(1970, 1, 1, tzinfo=timezone.utc)
-    date_obj = datetime.strptime(date, formatter)
+    date_obj = datetime.strptime(date, formatter) if formatter else dateparser.parse(date)
     return int(date_obj.strftime('%s') if date_obj.tzinfo is None else (date_obj - epoch).total_seconds())
 
 
 def main():
     args = demisto.args()
-    date_value = args['value']
-    formatter = args['formatter']
+    date_value = args.get('value')
```

Comment:
value is mandatory so we can keep it as is.
```suggestion
    date_value = args['value']
```